src/api.js

```diff
@@ -1,57 +1,51 @@
+
+import { initializeApp } from "firebase/app";
+import { getFirestore, collection, getDocs, doc, getDoc, query, where } from "firebase/firestore/lite"
+const apiKey = process.env.REACT_APP_API_KEY
+const authDomain = process.env.REACT_APP_AUTH_DOMAIN
+const projectId = process.env.REACT_APP_PROJECT_ID
+const storageBucket = process.env.REACT_APP_STORAGE_BUCKET
+const messagingSenderId = process.env.REACT_APP_MESSAGING_SENDER_ID
+const appId = process.env.REACT_APP_APP_ID
+const firebaseConfig = {
+  apiKey,
+  authDomain,
+  projectId,
+  storageBucket,
+  messagingSenderId,
+  appId
+};
+
+// Initialize Firebase
+const app = initializeApp(firebaseConfig);
+const db = getFirestore(app)
```

Comment:
You can remove this firebase configuration outside this file and put them to another file. Because in the future you can need to use it in different files.